Public/Src/Cache/ContentStore/Distributed/Stores/DistributedContentCopier.cs

```diff
@@ -135,7 +135,6 @@ protected override Task<BoolResult> ShutdownCoreAsync(OperationContext context)
                 int attemptCount = 0;
                 while (attemptCount < _retryIntervals.Count && (putResult == null || !putResult))
                 {
-                    Tracer.Debug(operationContext, $"Attempt #{attemptCount}: Copying {hashInfo.ContentHash} with {hashInfo.Locations.Count} locations");

```

Comment:
Please leave this, but outside while loop. Yes, you can infer this number from Redis logs, but this component should still log enough information that we can see what it is doing even if sources of input data change or change their logging.